commands/team_e2e_test.go

```diff
@@ -451,9 +453,12 @@ func (s *MmctlE2ETestSuite) TestRestoreTeamsCmd() {
 		teamName := "non-existent-team"
 
 		err := restoreTeamsCmdF(c, &cobra.Command{}, []string{teamName})
-		s.Require().Nil(err)
+		var expected error
+		errMessage := "unable to find team '" + teamName + "'"
+		expected = multierror.Append(expected, errors.New(errMessage))
 
-		errMessage := "Unable to find team '" + teamName + "'"
+		s.Require().NotNil(err)
```

Comment:
It's not necessary to check for the error separately, the next line will do this check as well.

Reply:
Removed this check.